components/ILIAS/Certificate/classes/User/class.ilUserCertificateRepository.php

```diff
@@ -622,4 +640,127 @@ public function deleteUserCertificatesForObject(int $userId, int $obj_id): void
 
         $this->logger->debug(sprintf('END - Successfully deleted certificate for user("%s") in object (obj_id: %s)"', $userId, $obj_id));
     }
+
+    /**
+     * @return ilUserCertificate[]
+     * @var array{certificate_id: null|string, issue_date: null|DateTime, object: null|string, owner: null|string} $filter
+     */
+    public function fetchCertificatesForOverview(array $filter, ?Range $range = null): array
+    {
+        global $DIC;
+        $user = $DIC->user();
```

Comment:
Please don't use the `$DIC` here. Instead, pass the language as (new) argument to the function.